supplier/views.py

```diff
@@ -10,6 +13,34 @@ def get_supplier(user):
         return False
 
 
+def receive_product(request, supplier):
```

Comment:
```suggestion
def get_supplier_product_by_name(request, supplier):
```

Reply:
i dont mind changing it, but if so it will be by product name and supplier (if we want to be exact).
so `def get_supplier_product_by_name_and_supplier`?

Reply:
`get_supplier_product_by_name_and_supplier` or `get_product_by_name_and_supplier` - both are good.